Client/Frontend/Popup/PaymentHandlerPopupView.swift

```diff
@@ -0,0 +1,284 @@
+        
+        if let inputType = inputType {
+            textField = UITextField(frame: CGRect.zero).then {
+                $0.keyboardType = inputType
+                $0.textColor = .black
+                $0.placeholder = inputPlaceholder ?? ""
+                $0.autocorrectionType = .no
+                $0.autocapitalizationType = .none
+                $0.layer.cornerRadius = 4
+                $0.layer.borderColor = UIColor(white: 0, alpha: 0.3).cgColor
+                $0.layer.borderWidth = 1
+                $0.delegate = self
+                $0.textAlignment = .center
+                $0.isSecureTextEntry = secureInput
+                containerView.addSubview($0)
+            }
+        }
+        
+        updateSubviews()
+        
+        setPopupContentView(view: containerView)
+        setStyle(popupStyle: .dialog)
+        setDialogColor(color: BraveUX.popupDialogColorLight)
+    }
+            
+    func addDisplayItemLabel(message: String) {
+        let displayItem = UILabel(frame: CGRect.zero)
+        displayItem.textColor = BraveUX.greyH
+        displayItem.textAlignment = .left
+        displayItem.font = UIFont.systemFont(ofSize: 12, weight: UIFont.Weight.regular)
+        displayItem.text = message
+        displayItem.numberOfLines = 0
+        
+        containerView.addSubview(displayItem)
+        displayItems.append(displayItem)
+        
+        updateSubviews()
+    }
+    
+    func addTotalLabel(message: String) {
+        totalItem = UILabel(frame: CGRect.zero)
+        totalItem.textColor = BraveUX.greyH
+        totalItem.textAlignment = .left
+        totalItem.font = UIFont.systemFont(ofSize: 12, weight: UIFont.Weight.regular)
+        totalItem.text = message
+        totalItem.numberOfLines = 0
+        
+        containerView.addSubview(totalItem)
+        updateSubviews()
+    }
+    
+    func clearDisplayItems() {
+        displayItems.removeAll()
+    }
+    
+    func update(title: String) {
+        titleLabel.text = title
+    }
+    
+    func clearTextField() {
+        textField?.text = nil
+    }
+    
+    func updateSubviews() {
+        titleLabel.adjustsFontSizeToFitWidth = false
+        messageLabel.adjustsFontSizeToFitWidth = false
+        displayItemTitle.adjustsFontSizeToFitWidth = false
+        totalItemTitle.adjustsFontSizeToFitWidth = false
+        
+        for label in displayItems {
+            label.adjustsFontSizeToFitWidth = false
+        }
+
+        updateSubviews(resizePercentage: 1.0)
+        
+        let paddingHeight = padding * 3.0
+        let externalContentHeight = dialogButtons.count == 0 ? paddingHeight : kPopupDialogButtonHeight + paddingHeight
+        let desiredHeight = UIScreen.main.bounds.height - externalContentHeight
+        
+        if containerView.frame.height > desiredHeight {
+            let resizePercentage = desiredHeight / containerView.frame.height
+            titleLabel.adjustsFontSizeToFitWidth = true
```

Comment:
Wrap this in an execution block, and just pass in `true`/`false` for all of these:

Pseudo-code (from Brandon's comment):
```
let updateFontAdjustment = { autoAdjust in
  labels = [titleLabel, messageLabel, displayItemTitle, totalItemTitle] + displayItems
  labels.forEach { $0.adjustsFontSizeToFitWidth = autoAdjust }
}
```

...

`updateFontAdjustments(true)`